src/maintainer/updating_pkgs.rst

```diff
@@ -187,6 +187,17 @@ Note that for long build logs one can do
 
 to save it in a text file for future inspection.
 
+Downloading prebuilt packages
+----------------
+
+To download prebuilt packages follow the steps below:
```

Comment:
I think a little bit more of discussion about when this should be used, for the reader who never attempted this before, would be amazing.